tests/unittests/test_util.py

```diff
@@ -956,6 +956,12 @@ def test_get_proc_ppid(self):
         self.assertEqual(my_ppid, util.get_proc_ppid(my_pid))
 
 
+@mock.patch('os.uname')
```

Comment:
The tests after this one don't follow our guidelines for unit tests: per the first bullet in https://cloudinit.readthedocs.io/en/latest/topics/hacking.html#unit-testing, we don't use function-level tests. Can we wrap this in a class, please?

(I'll submit a patch to fix the other tests here once this has landed, to avoid conflicts.)